.github/labeler.yml

```diff
@@ -0,0 +1,30 @@
+- name: Labeler
+  uses: actions/labeler@v3
```

Comment:
According to [Labeler readme](https://github.com/marketplace/actions/labeler),
it seems that the first 2 lines should be deleted.

```suggestion
```